app/bundles/InstallBundle/Configurator/Step/CheckStep.php

```diff
@@ -164,20 +164,8 @@ public function checkRequirements()
             $messages[] = 'mautic.install.extension.mbstring';
         }
 
-        if (extension_loaded('eaccelerator') && ini_get('eaccelerator.enable')) {
-            $messages[] = 'mautic.install.extension.eaccelerator';
-        }
-
-        if (function_exists('apc_store') && ini_get('apc.enabled')) {
-            if (!version_compare(phpversion('apc'), '3.1.13', '>=')) {
-                $messages[] = 'mautic.install.apc.version';
-            }
-        }
-
-        if (extension_loaded('suhosin')) {
-            if (stripos(ini_get('suhosin.executor.include.whitelist'), 'phar')) {
-                $messages[] = 'mautic.install.suhosin.whitelist';
-            }
+        if (!extension_loaded('apcu') || !ini_get('apc.enabled')) {
```

Comment:
This changes the core to _require_ installation of APCu instead of making it optional. Moving this check to the optional section. Previously we required the user to be on a specific version _if_ they had apc enabled.